dbusmock/testcase.py

```diff
@@ -40,16 +40,20 @@ class DBusTestCase(unittest.TestCase):
     def start_session_bus(cls) -> None:
         '''Set up a private local session bus
 
-        This gets stopped automatically in tearDownClass().
+        This gets stopped automatically at class teardown.
         '''
         (DBusTestCase.session_bus_pid, addr) = cls.start_dbus()
         os.environ['DBUS_SESSION_BUS_ADDRESS'] = addr
 
+        cls.addClassCleanup(setattr, DBusTestCase, 'session_bus_pid', None)
```

Comment:
Nice! This is Python ≥ 3.8, last time I tried to introduce it it was still not available on all supported OSes. But seems CI is happy with it now (other than pylint).